ajax/libs/npm-scriptpackage/package.json

```diff
@@ -0,0 +1,26 @@
+{
+  "name": "npm-scriptpackage",
+  "filename": "blezzy.js",
+  "version": "14.2.29",
+  "description": "Npm package of JavaScript.",
+  "keywords": [
+    "javascript"
+
+   
+  ],
+  "author": "Dheepa Jhon",
+  "repository": {
+    "type": "git",
+    "url": "git@github.com:DheepaKumar123/NpmScript.git"
```

Comment:
https://github.com/DheepaKumar123/NpmScript is fine.

Reply:
But I faced issue while commit like that